.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,50 @@
+name: Deploy
+
+on:
+  push:
+    branches: [master]
+
+# Deploy the Design System to production when the master branch is changed
+# Github Actions is not involved in deploying PR or branch previews – these are handled by Netlify
+
+jobs:
+  deploy:
+    needs: run-build
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Install the CF CLI
+        run: |
+          wget https://s3-us-west-1.amazonaws.com/v7-cf-cli-releases/releases/v7.2.0/cf7-cli-installer_7.2.0_x86-64.deb
+          sudo dpkg -i cf7-cli-installer_7.2.0_x86-64.deb
+
+      - name: Authenticate
+        env:
+          CF_API: "https://api.cloud.service.gov.uk"
+          CF_ORG: "govuk-design-system"
+          CF_SPACE: "design-system"
+          CF_USERNAME: "design-system-deploy+design-system@digital.cabinet-office.gov.uk"
+          CF_PASSWORD: ${{ secrets.CF_PASSWORD }}
```

Comment:
I wonder if we should be using an [environment](https://docs.github.com/en/free-pro-team@latest/actions/reference/environments) to make this secret only available to this job? 🕵🏻

Reply:
@36degrees I'm not sure how environments work - would we want to configure any protection rules? If not, I'm not sure the difference between referencing secrets vs environment secrets?